AirCasting/MobileSessions/MobileBluetoothSessionManager.swift

```diff
@@ -0,0 +1,64 @@
+// Created by Lunar on 30/04/2021.
+//
+
+import Foundation
+import CoreData
+
+class MobileBluetoothSessionManager {
+    
+    private let managedObjectContext: NSManagedObjectContext
+    private var locationProvider = LocationProvider()
+    var session: Session
+    
+    init(session: Session, managedObjectContext: NSManagedObjectContext) {
+        self.session = session
+        self.managedObjectContext = managedObjectContext
+    }
+    
+    func saveStream(_ stream: ABMeasurementStream) {
+        var sensorName = stream.sensorName
+        var sessionStream: MeasurementStream?
+        
+        switch sensorName {
+        case "AirBeam3-PM2.5":
+            sessionStream = session.pm2Stream ?? nil
+        case "AirBeam3-PM10":
+            sessionStream = session.pm10Stream ?? nil
+        case "AirBeam3-PM1":
+            sessionStream = session.pm1Stream ?? nil
+        case "AirBeam3-F":
+            sessionStream = session.FStream ?? nil
+        case "AirBeam3-RH":
+            sessionStream = session.HStream ?? nil
+        default:
+            Log.warning("AirBeam returned unknown measurement stream")
+        }
+        
+        if sessionStream != nil {
+            try! createMeasurement(value: stream.measuredValue, stream: sessionStream!)
+        }
+        
+    }
+    
+    func obtainCurrentLocation() -> CLLocationCoordinate2D {
+        locationProvider.currentLocation?.coordinate ?? CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
```

Comment:
Unfortunately, we have to have a fake location to be able to configure AirBeam. 
You can create a fixed "indoor" session which means you don't want to share a location, but AirBeam still needs it to be configured. We're using `CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)` to be consistent with Ania's (different Ania) solution in Android app. 

<img width="323" alt="Screenshot 2021-05-06 at 14 17 43" src="https://user-images.githubusercontent.com/46443841/117296871-d4d3f780-ae75-11eb-976c-46f5a8a47c26.png">

Reply:
@smialko we are not going to share the local database file with the android as some do so we can go with simplification / clarification of this logic. Having a fake location stored in the database can lead to multiple errors. for example crash on when trying to display it on the map. This will also require explanation for each curious developer why is this invalid value. filtering out value equal to 200/200 etc. nil is exactly the state wha this is representing. there was no location at this point. 
If the server needs it (which IMO is not a valid approach) we can fallback to this strange location while sending it to the server.

Reply:
The fallback location also if needed to be send to airbeam configuration is also a please where we can fallback to it. But buy default it is DEFINITELY more understandable to have it nil.